addons/web_dashboard/static/src/xml/dashboard.xml

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<templates xml:space="preserve">
+    <t t-name="DashboardMain">
+        <div class="o_web_dashboard">
```

Comment:
It is creating this kind of behavior.
On higher resolution it leave lot's of extra space 
1st one is with container
![download 9](https://cloud.githubusercontent.com/assets/7629089/9521961/818f981a-4cef-11e5-9871-ce69d8da6d47.png)